compiler/src/language_server/protocol.re

```diff
@@ -221,6 +221,26 @@ let response = (~id=?, result) => {
   flush(stdout);
 };
 
+let null_response = id => {
```

Comment:
```suggestion
let empty_response = id => {
```

Should we call this `empty_response`? More importantly, should we just rework `response` to take and Optional value instead of forcing the `Some(result)`? I think that would probably be the best solution overall.

Reply:
I'll rename it.  I made it its own function as id is mandatory for the empty response and currently optional for the regular response.

Reply:
Makes sense! Thanks for the clarification about the ID